src/state/api.js

```diff
@@ -28,26 +29,17 @@ export async function getProbeData(params, token) {
     },
     body: JSON.stringify({ query: params }),
   }).then(async (response) => {
-    // catch fetch error responses and surface them into
-    // the frontend.
-    if (response.status === 404) {
-      const e = new Error(FETCH_ERROR_MESSAGES.code404);
-      e.moreInformation =
-        "You may try changing the probe's current data filters.";
-      throw e;
-    } else if (response.status >= 400 && response.status < 600) {
-      const msg =
-        response.status < 500
-          ? FETCH_ERROR_MESSAGES.code4xx
-          : FETCH_ERROR_MESSAGES.code5xx;
-      const e = new Error(msg);
-      const txt = await response.text();
-      e.moreInformation = `GLAM encountered a problem trying to show ${
-        params.probe
-      }, so we'll
-      report the error here:
-      ${txt.split('\n').slice(0, 2)}`;
-      throw e;
+    // Catch fetch error responses and show them in the UI.
+    if (response.status >= 400 && response.status < 600) {
+      const errorCode = 'code' + response.status;
```

Comment:
```
  34:25  error  Unexpected string concatenation  prefer-template
```

Reply:
That's a little strict, simple concatenations shouldn't need it. Don't mind changing it just weird.

Reply:
Yeah, Airbnb rules. 🤷 

We could change that rule if we prefer.

Reply:
I do prefer string templating in cases like this